test/utils/shared.ts

```diff
@@ -22,3 +25,50 @@
 export function generateUniqueName(baseName = "", length = 4): string {
   return `${baseName}${crypto.randomBytes(length).toString("hex")}`;
 }
+
+export async function createDatabase(frame: Frame) {
+  const dbId = generateUniqueName("db");
+  const collectionId = generateUniqueName("col");
+  const sharedKey = `${generateUniqueName()}`;
+  // create new collection
+  await frame.waitFor('button[data-test="New Collection"]', { visible: true });
```

Comment:
>await frame.waitFor('button[data-test="New Collection"]' [](start = 2, length = 56)

can we use the button on command bar instead of the splash page.

Reply:
i verified that this clicks on the new collection button on the command bar